plugins/modules/mso_schema_template_contract_service_graph.py

```diff
@@ -17,6 +17,9 @@
 description:
 - Manage the service graph association with a contract in schema template on Cisco ACI Multi-Site.
 - The Contract Service Graph parameter is supported on versions of MSO/NDO that are 3.3 or greater.
+- The recommended approach is to use M(cisco.mso.mso_schema_template_contract_service_graph) rather than the Contract Service Graph.
```

Comment:
Shouldn't this be:
```suggestion
- The recommended approach is to use M(cisco.mso.ndo_schema_template_contract_service_chain) rather than the Contract Service Graph.
```